src/admin-display.cls.php

```diff
@@ -46,6 +46,65 @@ class Admin_Display extends Base {
 
 	private $_btn_i = 0;
 
+	/*
+	 * List of settings with filters and return type.
+	 * 
+	 * @since  7.4
+	 */
+	protected static $settings_filters  = array(
+		// Crawler - Blocklist
+		'crawler-blocklist' => [
```

Comment:
keep in mind short array tag will not pass format checker

Reply:
@timotei-litespeed Actually short tag is easier to read and doesn't hurt. I will update the rule to allow it.